gaia-list.js

```diff
@@ -256,15 +256,6 @@ module.exports = component.register('gaia-list', {
       width: auto;
     }
 
-    /**
-     * Reverse the icons when the document is RTL mode
-     */
-
-    :host-context([dir=rtl]) ::content i:before {
-      transform: scale(-1, 1);
-      text-align: end;
-    }
-
```

Comment:
(our two comments crosses in the air apparently)

if you don't mind, I would prefer:
- keep in the html of gaia-list the data-icon="forward/back"
- add in gaia-list.js something like
```css
 :host-context([dir=ltr]) ::content i[data-icon="forward"] {
  content: 'right';
} 
 :host-context([dir=rtl]) ::content i[data-icon="forward"] {
  content: 'left';
} 
```
That's what I meant by direction specific rules.

The main point of mirroring icons in gaia-icons was to avoid this scale(-1) which is far from free performance-wise (especially for gaia-list and the forward icon because they will be a lot of them!).

This is a bit more verbose, but:
- we keep the functionality
- imho we capture perfectly the semantic of arrow in bidi context: "forward is right in LTR and left in RTL". WDYT?